src/main/java/seedu/address/commons/core/GuiSettings.java

```diff
@@ -73,7 +83,8 @@ public String toString() {
         StringBuilder sb = new StringBuilder();
         sb.append("Width : " + windowWidth + "\n");
         sb.append("Height : " + windowHeight + "\n");
-        sb.append("Position : " + windowCoordinates);
+        sb.append("Position : " + windowCoordinates + "\n");
+        // sb.append("Theme: " + uiTheme);
```

Comment:
Perhaps you mean to append the themeName here?

Reply:
Oh, yeah, thank you for pointing that out